### DISCUSSION TITLE
Welcome to freshbey Discussions!

### DISCUSSION THREAD

**Discussion by roopeshsn** [roopeshsn/freshbey#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avinashji1302** [comment]
Hii I am Avinash and I am really looking for contribution and never contributed in project so far and i really want to contribute could please guide me or assign me any kind of work?

**Reply by roopeshsn** [reply]
Hi @avinashji1302

Try out the application in your local machine by forking and cloning the repository. Go through the source code and understand how things are implemented. You can develop a new feature to implement or improve the existing one.
